Filter/ModelFilter.php

```diff
@@ -120,7 +127,14 @@ protected function handleModel(ProxyQueryInterface $queryBuilder, $alias, $data)
         $parameterName = $this->getNewParameterName($queryBuilder);
 
         if (isset($data['type']) && $data['type'] == EqualType::TYPE_IS_NOT_EQUAL) {
-            $this->applyWhere($queryBuilder, sprintf('%s != :%s', $alias, $parameterName));
+            $or = $queryBuilder->expr()->orX();
+
+            $or->add($queryBuilder->expr()->neq($alias, ':'.$parameterName));
```

Comment:
How about refactoring this into a private function? You can use `notIn` here too, right?